.claude/architecture.md

```
@@ -0,0 +1,37 @@
+# Architecture
+
+## Core
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)